python/GangaLHCb/Lib/GaudiRunApp/GaudiRunApp.py

```diff
@@ -57,40 +65,62 @@ class GaudiRun(IPrepareApp):
 
     The actual command run on the WN is:
 
-        ./run gaudirun.py myOptsFile.py data.py
+        ./run gaudirun.py optionsFile.py data.py
 
-    If you would prefer to have your optsfile run as a python application then set 'job.application.runWithPython = True'
-    This then changes the command run on the WN to be:
+    If you would prefer to have your optsfile run as a python application then set 'job.application.useGaudiRun = False'
+    This then changes the command run on the WN to bu:
 
         ./run python OptsFileWrapper.py
 
-        Here the OptsFileWrapper script imports the data.py describing the data to be run over and executes myOpts with 'execfile'
+        Here the OptsFileWrapper script imports the data.py describing the data to be run over and executes options with 'execfile'
 
     """
     _schema = Schema(Version(1, 0), {
         # Options created for constructing/submitting this app
-        'directory':    SimpleItem(preparable=1, defvalue=None, typelist=[None, str], comparable=1,
-            doc='A path to the project that you\'re wanting to run.'),
-        'build_opts':   SimpleItem(defvalue=[""], typelist=[str], sequence=1, strict_sequence=0,
-            doc="Options to be passed to 'make ganga-input-sandbox'"),
-        'myOpts':       GangaFileItem(defvalue=None, doc='File which contains the extra opts I want to pass to gaudirun.py'),
-        'uploadedInput': GangaFileItem(defvalue=None, doc='This stores the input for the job which has been pre-uploaded so that it gets to the WN'),
-        'runWithPython':SimpleItem(defvalue=False, doc='Should \'myOpts\' be run as "python myOpts.py data.py" rather than "gaudirun.py myOpts.py data.py"'),
+        'directory':    SimpleItem(defvalue=None, typelist=[None, str], comparable=1, doc='A path to the project that you\'re wanting to run.'),
+        'options':       GangaFileItem(defvalue=None, doc='File which contains the extra opts I want to pass to gaudirun.py'),
```

Comment:
Remove the word "extra" from the documentation as it is confusing with the `extraopts` below.